opensrp-reveal/src/main/java/org/smartregister/reveal/fragment/TaskRegisterFragment.java

```diff
@@ -73,6 +74,8 @@
 
     private CardView indicatorsCardView;
 
+    private TextView filterTextView;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'filterTextView' could be replaced by a local variable.](https://app.codacy.com/app/OpenSRP/opensrp-client-reveal/pullRequest?prid=4684568)